rero_ils/modules/items/views/api_views.py

```
@@ -131,48 +144,115 @@ def do_item_jsonify_action(func):
         except CirculationException as error:
-            abort(403, error.description or str(error))
+            _reindex_item_if_needed(item, removed_temp_item_type_name)
+            # Circulation exceptions (e.g., patron blocked, checkout limit reached)
+            # Return JSON response with status code and message to allow including
+            # additional information like removed_temporary_item_type
+            error_response = _add_removed_item_type_info(
+                {"status": 403, "message": error.description or str(error)}, removed_temp_item_type_name
+            )
+            return jsonify(error_response), 403
         except NotFound as error:
             raise error
```

Ruff (0.14.13)

Use `raise` without specifying exception name

Remove exception name

(TRY201)